test/integration_test.rb

```diff
@@ -67,6 +67,15 @@ def test_isolated_with_fail_fast
         ENV.delete('ISOLATED')
         ENV.delete('FAIL_FAST')
       end
+
+      def test_select_untracked_files
+        FileUtils.mkdir('test')
+        fizz_test = './fizz_test.rb'
+        File.write(fizz_test, File.read(fizz_test).gsub("require_relative '\.", "require_relative '.."))
+        FileUtils.mv('./fizz_test.rb', './test/fizz_test.rb')
```

Comment:
Nit-picky bu since you're assigning a variable for this file's name, you can re-use it here.